modules/administration/pages/monitoring.adoc

```diff
@@ -214,6 +208,31 @@ image::monitoring_configure_formula.png[scaledwidth=80%]
 
 
 
+.Procedure: Enable Service Discovery
+
+[IMPORTANT]
+====
+This feature is a technical preview available in {productname} 4.0.2 and later.
+It should not be used in production systems.
+====
+
+. Open the Prometheus static configuration file at [path]``/etc/prometheus/prometheus.yml``.
```

Comment:
Maybe we should make it really clear here that this needs to be done on the machine where Prometheus is installed and running?